CONTRIBUTING.md

```diff
@@ -107,7 +121,47 @@ just ready
 This command will run the same commands of the CI: format, lint, tests and code generation.
 Eventually everything should be "green" 🟢 and commit all the code that was generated.
 
-## Language Server and VS Code Extension Development
+#### Generated files
+
+If you work on some parser and you create new nodes or modify existing ones, will need to run a command to update some files that are auto-generated.
+
+##### `cargo codegen grammar`
+
+This command will update the syntax of the parsers.
+
+The source is generated from the [`ungram` files](https://github.com/biomejs/biome/blob/main/xtask/codegen/js.ungram).
+
+##### `cargo codegen test`
+
+This command will create new tests for your parser. We currently have a neat infrastructure
+where tests for parser are generated com inline comments found inside
+the source code.
```

Comment:
We are moving away from this approach. While the JS and JSON parsers still use this infra, the CSS parser now uses a more straightforward process. Maybe we should rephrase this paragraph to communicate this to the contributors.